### ISSUE TITLE
min_power and max_power from input field

### ISSUE THREAD

**Issue by nebula7219** [patrickcollins12/esphome-fan-controller#32]
Hi
How can I get min_power and max_power from an input_number helper ? I want to change the max and min power at night
Thx
Ralph

**Comment by patrickcollins12** [comment]
Unfortunately, those variables can't be set or get in real time only on setup.

Esp32 time can be synced to home assistant. Then you can use the time of day to cap the number in a template output.

**Comment by patrickcollins12** [comment]
Setting the time: https://esphome.io/components/time/homeassistant.html
Altering the power output: https://esphome.io/components/output/#output-set-level-action

**Comment by nebula7219** [comment]
thanks for the info, that's a good solution. is it possible to use a button for restart and the new min/max_power from an input_number helper? if that's possible, can you help me with the code? sorry for my bad English, I speak German

**Comment by patrickcollins12** [comment]
Yes you could try that. Try writing some lambdas that stitch those three concepts together

**Comment by nebula7219** [comment]
I tried it but it doesn't work, I don't think my skills are good enough.
I tried this:
`esphome:
  name: fan_pid_controller
  on_boot: 
      then:
        - output.set_level_min_power:
          id: console_fan_speed
          level: !lambda |-
          return id(abluft_min.state) / 100.0;
`
